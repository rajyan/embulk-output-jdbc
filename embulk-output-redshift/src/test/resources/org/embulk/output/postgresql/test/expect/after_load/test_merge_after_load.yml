table: test1
mode: merge
after_load: "update test1 set varchar_item = 'x' where int_item = 9"
