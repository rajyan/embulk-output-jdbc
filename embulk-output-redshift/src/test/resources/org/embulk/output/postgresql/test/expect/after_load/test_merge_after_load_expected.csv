A001,9,x
A002,0,b
A003,9,x
B001,0,z
B002,9,x
